eng/pipelines/common/global-build-job.yml

```diff
@@ -206,15 +206,25 @@ jobs:
           df -h
         displayName: Disk Usage before Build
 
-      # Install Wasm dependencies: emscripten, LLVM, NodeJS
-    - ${{ if and(eq(parameters.runtimeFlavor, 'coreclr'), eq(parameters.archType, 'wasm')) }}:
+      # Install Wasm dependencies on Windows: emscripten, LLVM, NodeJS
+    - ${{ if and(eq(parameters.hostedOs, 'windows'), and(eq(parameters.runtimeFlavor, 'coreclr'), eq(parameters.archType, 'wasm'))) }}:
       - script: call $(Build.SourcesDirectory)/eng/pipelines/runtimelab/install-emscripten.cmd $(Build.SourcesDirectory)\wasm-tools
         displayName: Install/activate emscripten
       - script: call $(Build.SourcesDirectory)/eng/pipelines/runtimelab/install-llvm.cmd $(Build.SourcesDirectory)\wasm-tools $(Build.SourcesDirectory) ${{ parameters.buildConfig }}
         displayName: Install/build LLVM
       - script: call $(Build.SourcesDirectory)/eng/pipelines/runtimelab/install-nodejs.cmd $(Build.SourcesDirectory)\wasm-tools
         displayName: Install NodeJS
 
+      # Install Wasm dependencies on Linux: emscripten, LLVM, NodeJS.
+    - ${{ if and(eq(parameters.hostedOs, 'linux'), and(eq(parameters.runtimeFlavor, 'coreclr'), eq(parameters.archType, 'wasm'))) }}:
+      - script: $(Build.SourcesDirectory)/eng/pipelines/runtimelab/install-emscripten.sh $(Build.SourcesDirectory)/wasm-tools
+        displayName: Install/activate emscripten
+      - script: pwsh $(Build.SourcesDirectory)/eng/pipelines/runtimelab/install-llvm.ps1 -Configs ${{ parameters.buildConfig }} -CI
```

Comment:
`parameters.buildConfig` can be `Checked`, to the normalization of of `Config` in `install-llvm.cmd` needs to be moved into `install-llvm.ps1`.

Reply:
Done